vsphere_server_plugin/server.py

```diff
@@ -643,14 +656,13 @@ def get_state(ctx, server_client, server, networking, os_family):
             )
 
         ctx.instance.runtime_properties[NETWORKS] = nets
-        try:
-            ctx.instance.runtime_properties[IP] = (
-                manager_network_ip or
-                get_ip_from_vsphere_nic_ips(server_obj.guest.net[0])
-            )
-        except IndexError:
+        ctx.instance.runtime_properties[IP] = manager_network_ip or default_ip
+        if not ctx.instance.runtime_properties[IP]:
```

Comment:
Is this property always set? Wouldn't it be safer to run `ctx.instance.runtime_properties.get(IP)`?

Reply:
It set in previous line:
```python
ctx.instance.runtime_properties[IP] = ....
if not ctx.instance.runtime_properties[IP]:
```